addons/project/static/src/actions/client_actions.js

```diff
@@ -146,3 +147,24 @@ registry
         });
         return params.next;
     });
+
+// Update the embedded settings of a project for a given action and user settings
+registry.category("actions").add("update_project_embedded_settings", async (env, action) => {
+    const params = action.params || {};
+    const embeddedSettings = await env.services.orm.call(
```

Comment:
those new settings could be directly given in params, this would remove that rpc roundtrip

Reply:
I don't think it's possible

<img width="1100" height="126" alt="Image" src="https://github.com/user-attachments/assets/82764afa-de07-40cd-8dc3-3fac120da707" />